packages/core/src/tools/web-fetch.ts

```diff
@@ -395,7 +395,7 @@ export class WebFetchTool extends BaseDeclarativeTool<
         properties: {
           prompt: {
             description:
-              'A comprehensive prompt that includes the URL(s) (up to 20) to fetch and specific instructions on how to process their content (e.g., "Summarize https://example.com/article and extract key points from https://another.com/data"). Must contain as least one URL starting with http:// or https://.',
+              'A comprehensive prompt that includes the URL(s) (up to 20) to fetch and specific instructions on how to process their content (e.g., "Summarize https://example.com/article and extract key points from https://another.com/data"). All URLs to be fetched must be valid, complete, and start with http:// or https://.',
```

Comment:
can we say that a valid URL start with http:// or http:// and end with .com, .org. .edu or any domain name such as ths?

I think when we say that URL should be valid and complete, we are not defining what is valid and complete. Lets specify that as well as mentioned above.
Also do we want to remove the specification that a prompt should have at least 1 URL?